### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by APMichael**
- Minor correction.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [1.14.8 / 5.69.8] - 2024-09-09
 
 ### Changed
-- allowed users to import/export boxes with .zip files.
+- allow users to import/export boxes with .zip files
 
 ### Fixed
 - fixed a cert issue
```
Comment (isaak654) [inline CHANGELOG.md:12]:
```suggestion
- fixed a supporter certificate issue
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — CHANGES_REQUESTED** [review]
https://github.com/sandboxie-plus/Sandboxie/blob/1a62e44b88f58c5e587b07d52b8fd44f2287b0e1/CHANGELOG.md#added-5

>   - use "UseChangeSpeed=y" to open this feature, use "AddTickSpeed" / "AddSleepSpeed" / "AddTimerSpeed" / "LowTickSpeed" / "LowSleepSpeed" / "LowTimerSpeed" (Number) to set

I don't understand the meaning, I think it can be better explained.

**Comment by APMichael** [comment]
> I do not understand the meaning, I think it can be better explained.

I have never used this function either. Someone who uses the function more often would have to provide a better explanation.

**Comment by isaak654** [comment]
According to David:

> UseChangeSpeed=y is the trigger to enable "AddTickSpeed", "AddSleepSpeed", "AddTimerSpeed", "LowTickSpeed", "LowSleepSpeed", "LowTimerSpeed".
>
> @love-code-yeyixiao added this feature and i asked him to provide more details on the target usecases, no reply yet. I can give a tehcnical description what it does but whats really usefull for is a different story, one can use ot to make games run faster or slower, influence video playback speed etc.

**Comment by love-code-yeyixiao** [comment]
Add/LowTickSpeed means&nbsp;speeding up or slowing down the frequency of timing triggers that are set using SetTimer over a period of time.
Add/LowSleepSpeed means lengthening or shortening the fixed length of time that the program intentionally delays by calling the Sleep series of functions.

Add/LowTickSpeed&nbsp;means that the results obtained by the program for the system running time interval between two operations can be referred to the change gear.
They all use multiples instead of adding or subtracting. Add series configuration by multiplication, Low series configuration by division, note that they both need to use integers.

---Original---
From: ***@***.***&gt;
Date: Fri, Sep 13, 2024 21:14 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] Update CHANGELOG.md (PR #4228)




 
According to David:
  
UseChangeSpeed=y is the trigger to enable "AddTickSpeed", "AddSleepSpeed", "AddTimerSpeed", "LowTickSpeed", "LowSleepSpeed", "LowTimerSpeed".
 
@love-code-yeyixiao added this feature and i asked him to provide more details on the target usecases, no reply yet. I can give a tehcnical description what it does but whats really usefull for is a different story, one can use ot to make games run faster or slower, influence video playback speed etc.

**Review by isaak654 — APPROVED** [review]

**Comment by bastik-1001** [comment]
It appears that there is no documentation for these features. Wouldn't it be smart to have some more details on something that appears to be rather complicated? This feature might not be for me, but I am not even sure how and when I would be using it. Maybe it's not for most users, but some may find it useful, not even knowing that the feature exist. It isn't `BoxNameTitle=n` where the label is obvious and it's an on/off switch.

**Comment by love-code-yeyixiao** [comment]
For example, you can skip the delay code that may extend the analysis time in hidden malicious software by adding SleepSpeed=0. This is only for code with fixed delay, so it will not interfere with the actual operation that needs time. For virus analysts, this is very useful, isn't it?




---Original---
From: ***@***.***&gt;
Date: Sat, Sep 14, 2024 06:58 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] Update CHANGELOG.md (PR #4228)




 
@isaak654 approved this pull request.

**Comment by love-code-yeyixiao** [comment]
I think it's OK.



---Original---
From: ***@***.***&gt;
Date: Sat, Sep 14, 2024 19:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] Update CHANGELOG.md (PR #4228)




 
@isaak654 commented on this pull request.
 
 
In CHANGELOG.md:
 &gt; +- added option to limit the memory of sandboxed processes and the number of processes in a single sandbox through job object (thanks Yeyixiao) +  - use "TotalMemoryLimit" (number, in bytes) to set the overall memory limit for the sandbox, and "ProcessMemoryLimit" (number, in bytes) to limit memory for individual processes +  - use "ProcessNumberLimit" (number) to set process number limit +- added ability to adjust the logic speed of sandboxed processes, including reduced fixed latency and modified single-player speed (thanks Yeyixiao) +  - Note: you can set "UseChangeSpeed=y" to configure the following options: "AddTickSpeed", "AddSleepSpeed", "AddTimerSpeed", "LowTickSpeed", "LowSleepSpeed" and "LowTimerSpeed" (integer values only) +  - Note: these options use multiples instead of adding or subtracting; the "Add" series is configured by multiplication, while the "Low" series by division +  - Note: when set to "AddSleepSpeed=0", all sleep function calls will be skipped  ⬇️ Suggested change -- added option to limit the memory of sandboxed processes and the number of processes in a single sandbox through job object (thanks Yeyixiao) -  - use "TotalMemoryLimit" (number, in bytes) to set the overall memory limit for the sandbox, and "ProcessMemoryLimit" (number, in bytes) to limit memory for individual processes -  - use "ProcessNumberLimit" (number) to set process number limit -- added ability to adjust the logic speed of sandboxed processes, including reduced fixed latency and modified single-player speed (thanks Yeyixiao) -  - Note: you can set "UseChangeSpeed=y" to configure the following options: "AddTickSpeed", "AddSleepSpeed", "AddTimerSpeed", "LowTickSpeed", "LowSleepSpeed" and "LowTimerSpeed" (integer values only) -  - Note: these options use multiples instead of adding or subtracting; the "Add" series is configured by multiplication, while the "Low" series by division -  - Note: when set to "AddSleepSpeed=0", all sleep function calls will be skipped +- added option to limit the memory of sandboxed processes and the number of processes in a single sandbox through job object (thanks Yeyixiao) +  - use "TotalMemoryLimit" (number, in bytes) to set the overall memory limit for the sandbox, and "ProcessMemoryLimit" (number, in bytes) to limit memory for individual processes +  - use "ProcessNumberLimit" (number) to set process number limit +- added ability to adjust the logic speed of sandboxed processes, including reduced fixed latency and modified single-player speed (thanks Yeyixiao) +  - Note: you can set "UseChangeSpeed=y" to configure the following options: "AddTickSpeed", "AddSleepSpeed", "AddTimerSpeed", "LowTickSpeed", "LowSleepSpeed" and "LowTimerSpeed" (integer values only) +  - Note: these options use multiples instead of adding or subtracting; the "Add" series is configured by multiplication, while the "Low" series by division +  - Note: when set to "AddSleepSpeed=0", all sleep function calls will be skipped. For example, you can bypass fixed delay code in hidden malware, reducing analysis time without affecting essential operations. Useful for virus analysts.  
@love-code-yeyixiao Do you agree with the modified changelog notes from line 170 to line 176? If not, what would you like to change or add? Thank you.

**Pull request merged**